docs/how-to-use-monty/code-base-structure.md

```diff
@@ -10,10 +10,9 @@ The basic repository structure looks as follows:
 .
 |-- docs/                       # .md files for documentation
 |-- rfcs/                       # Merged RFCs
-|-- benchmarks/                 # experiments testing Monty
```

Comment:
I thought the benchmarks folder still existed, it just doesn't contain configs anymore?